app/controllers/admin/base_controller.rb

```diff
@@ -52,6 +52,10 @@ def forbid_editing_of_locked_documents
     end
   end
 
+  def preview_design_system_user?
+    current_user.has_permission? "Preview Design System"
```

Comment:
I think this needs to be "Preview design system". There's a convention in whitehall to lowercase permission strings (other than the first word) which i followed

Reply:
done